R/validation-type.R

```diff
@@ -65,8 +65,6 @@ isOfType <- function(object, type, nullAllowed = FALSE) {
 #' validateIsLogical(TRUE)
 #' @export
 validateIsOfType <- function(object, type, nullAllowed = FALSE) {
-  type <- c(type)
```

Comment:
This did nothing. Even after removing it, all tests pass, and even the "off label" behaviour is intact:

``` r
library(ospsuite.utils)

validateIsOfType(list(1, "x"), c("numeric", "character"))
#> NULL
validateIsOfType(list(1, "x"), list("numeric", "character"))
#> NULL
```

<sup>Created on 2022-03-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>